__tests__/__snapshots__/thesis-editor-test.js.snap

```diff
@@ -0,0 +1,151 @@
+exports[`test renders ThesisEditor 1`] = `
```

Comment:
This file is a jest "snapshot". It records similar to ruby's VCR gem, and displays any changes if they happen.